### PR TITLE
Misc code style clean up

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: common/src/Logging.ProvisioningTransport.Common.cs [hunk 1]
```diff
@@ -10,20 +10,21 @@ internal sealed partial class Logging : EventSource
...
-            TimeSpan? retryAfter, 
+            string attestationType,
+            string operationId,
+            TimeSpan? retryAfter,
             string status)
         {
             DebugValidateArg(thisOrContextObject);
             DebugValidateArg(attestationType);
             DebugValidateArg(retryAfter);
+
```
Comment (drwill-ms) [inline common/src/Logging.ProvisioningTransport.Common.cs:27]: I wonder if the `if (IsEnabled)` check shouldn't be before the DebugValidateArgs.

File: provisioning/device/src/ProvisioningTransportHandler.cs [hunk 2]
```diff
@@ -19,6 +18,11 @@ public abstract class ProvisioningTransportHandler : IDisposable
         private ProvisioningTransportHandler _innerHandler;
         private int _port;
 
+        /// <summary>
```
Comment (drwill-ms) [inline provisioning/device/src/ProvisioningTransportHandler.cs:21]: ctor above properties

File: provisioning/device/src/ProvisioningTransportHandler.cs [hunk 3]
```diff
@@ -92,6 +85,15 @@ public int Port
             return _innerHandler.RegisterAsync(message, cancellationToken);
         }
 
+        /// <summary>
+        /// Releases the unmanaged resources and disposes of the managed resources used by the invoker.
+        /// </summary>
+        public void Dispose()
```
Comment (drwill-ms) [inline provisioning/device/src/ProvisioningTransportHandler.cs:91]: public above protected/internal/private

File: provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs [hunk 4]
```diff
@@ -58,41 +55,28 @@ public ProvisioningTransportHandlerHttp()
...
-                else if (message.Security is SecurityProviderX509)
-                {
-                    authStrategy = new HttpAuthStrategyX509((SecurityProviderX509)message.Security);
-                }
-                else if (message.Security is SecurityProviderSymmetricKey)
-                {
-                    authStrategy = new HttpAuthStrategySymmetricKey((SecurityProviderSymmetricKey)message.Security);
-                }
-                else
+                switch (message.Security)
```
Comment (drwill-ms) [inline provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs:58]: This must be a new syntax. VS suggested it. Neat!

File: provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs [hunk 5]
```diff
@@ -105,130 +89,125 @@ public ProvisioningTransportHandlerHttp()
...
+                    httpClientHandler.UseProxy = Proxy != null;
                     httpClientHandler.Proxy = Proxy;
-                    if (Logging.IsEnabled)
-                    {
-                        Logging.Info(this, $"{nameof(RegisterAsync)} Setting HttpClientHandler.Proxy");
-                    }
+                    Logging.Info(this, $"{nameof(RegisterAsync)} Setting HttpClientHandler.Proxy");
                 }
 
+                var builder = new UriBuilder
```
Comment (drwill-ms) [inline provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs:97]: Moved this down to just before it was needed.

File: common/src/Logging.ProvisioningTransport.Common.cs (hunk 1 above)
Comment (abhipsaMisra) [reply common/src/Logging.ProvisioningTransport.Common.cs:27]: you are right, it should be

**Review by abhipsaMisra — APPROVED** [review]

**Review by azabbasi — APPROVED** [review]

File: provisioning/device/src/ProvisioningTransportHandler.cs [hunk 6]
```diff
@@ -19,6 +18,11 @@ public abstract class ProvisioningTransportHandler : IDisposable
         private ProvisioningTransportHandler _innerHandler;
         private int _port;
 
+        /// <summary>
+        /// Creates an instance of the ProvisioningTransportHandler class.
```
Comment (barustum) [inline provisioning/device/src/ProvisioningTransportHandler.cs:22]: nit: <see cref="ProvisioningTransportHandler"/>

**Review by barustum — APPROVED** [review]

**Pull request merged**